firstuseauthenticator/templates/reset.html

```diff
@@ -0,0 +1,38 @@
+{% extends "page.html" %}
+
+{% block main %}
+
+<div class="container">
+<form action="{{post_url}}" method="post" role="form">
```

Comment:
Can you indent this as well, so it's clearer when the container div starts and ends?